src/store/ducks/plans.ts

```diff
@@ -425,64 +425,67 @@ export const fetchPlanRecords = (planList: PlanRecordResponse[] = []): FetchPlan
 /** getPlansById - get plansById by intervention type
  * @param {Partial<Store>} state - the redux store
  * @param {InterventionType} intervention - the intervention type
- * @param {string[]} status - the plan status
+ * @param {string[]} statusList - the plan statuses
  * @param {string} reason - the plan reason
  */
 export function getPlansById(
   state: Partial<Store>,
   intervention: InterventionType = InterventionType.FI,
-  status: string[] = [PlanStatus.ACTIVE],
+  statusList: string[] = [PlanStatus.ACTIVE],
   reason: string | null = null
 ): { [key: string]: Plan } {
   const plansById = (state as any)[reducerName].plansById;
   return pickBy(
     plansById,
     (plan: Plan) =>
       plan.plan_intervention_type === intervention &&
-      (status.length ? status.includes(plan.plan_status) : true) &&
+      (statusList.length ? statusList.includes(plan.plan_status) : true) &&
       (reason ? plan.plan_fi_reason === reason : true)
   );
 }
 
 /** getPlansArray - get an array of Plans by intervention type
  * @param {Partial<Store>} state - the redux store
  * @param {InterventionType} intervention - the intervention type
- * @param {string[]} status - the plan status
+ * @param {string[]} statusList - a list of the desired the plan statuses
  * @param {string} reason - the plan reason
  * @param {string} parentJurisdictionId - The jurisdiction_parent_id of the plan
  * @param {string[]} jurisdictions - jurisdiction IDs list to find plan's jurisdiction_id within
  */
 export function getPlansArray(
   state: Partial<Store>,
   intervention: InterventionType = InterventionType.FI,
-  status: string[] = [],
+  statusList: string[] = [],
   reason: string | null = null,
   jurisdictions: string[] = [],
   parentJurisdictionId: string | null = null
 ): Plan[] {
   return values((state as any)[reducerName].plansById).filter(
     (plan: Plan) =>
       plan.plan_intervention_type === intervention &&
-      (status.length ? status.includes(plan.plan_status) : true) &&
+      (statusList.length ? statusList.includes(plan.plan_status) : true) &&
       (reason ? plan.plan_fi_reason === reason : true) &&
       (jurisdictions.length ? jurisdictions.includes(plan.jurisdiction_id) : true) &&
-      (parentJurisdictionId ? plan.jurisdiction_path.includes(parentJurisdictionId) : true)
+      (parentJurisdictionId && !plan.jurisdiction_path ? false : true) &&
```

Comment:
This is the main fix in the PR.  We are ensuring that we take into account the possibility of `plan.jurisdiction_path` being null